cli/src/print.rs

```diff
@@ -37,14 +37,16 @@ pub fn print_update_message() {
     eprintln!("{:-^50}\n\n", "");
 }
 
-pub fn print_sc_help(app: &mut Command, subcommand: &str) {
-    for sc in app.get_subcommands_mut() {
-        if sc.get_name() == subcommand {
-            let _ = sc.print_help();
-            break;
+pub fn print_sc_help(mut app: &mut Command, subcommands: &[&str]) {
+    for subcommand in subcommands {
+        match app.get_subcommands_mut().find(|sc| &sc.get_name() == subcommand) {
+            Some(subcommand) => app = subcommand,
+            // Subcommand doesn't exist; don't print anything.
+            None => return,
```

Comment:
Currently this relies on us always passing the right subcommands in, otherwise it will just silently exit the program. I think that's fine, but maybe we should print an error or crash just in case?

Reply:
I think some sort of panic would be reasonable here. If we ever hit that case, it definitely means a bug in the code.